### PR TITLE
Create html for order-section

### PR DESCRIPTION
**Pull request by dmeshkovGit**

### CODE REVIEW

File: src/partials/order.html [hunk 1]
```diff
@@ -1,3 +1,42 @@
-<section class="order">
-
-</section>
\ No newline at end of file
+<section class="order-section">
+  <div class="order-form container">
```
Comment (SergGrishkov) [inline src/partials/order.html]: After tag `section ` should be `<div class="order-container container">`. In container should be added all your code.

File: src/partials/order.html [hunk 2]
```diff
@@ -1,3 +1,42 @@
-<section class="order">
-
-</section>
\ No newline at end of file
+<section class="order-section">
```
Comment (SergGrishkov) [inline src/partials/order.html]: In section Order should be incluge **img** girl in green Vyshyvanka

Comment (SergGrishkov) [reply src/partials/order.html]: Should be add **id** in **section** `<section class="order" id="order">`

**Review by SergGrishkov — CHANGES_REQUESTED** [review]

File: src/partials/order.html [hunk 3]
```diff
@@ -1,3 +1,46 @@
-<section class="order">
-
-</section>
\ No newline at end of file
+<section class="order-section" id="order-section">
```
Comment (SergGrishkov) [inline src/partials/order.html]: `-section` should be remove

File: src/partials/order.html [hunk 4]
```diff
@@ -1,3 +1,46 @@
...
+          required
+        />
+        <textarea
+          class="order-form-comment"
+          name="user-comments"
+          title="Your-comment"
+          placeholder="Comments"
+        ></textarea>
+        <button class="order-form-btn" type="submit">
+          Send<svg><use href="../img/header/icons.svg"></use></svg>
```
Comment (SergGrishkov) [inline src/partials/order.html]: 1 dot

File: src/partials/order.html [hunk 5]
```diff
@@ -1,3 +1,46 @@
...
+        </button>
+      </form>
+    </div>
+    <img
+      src="../img/your-order/content/order-product-tablet.jpg"
```
Comment (SergGrishkov) [inline src/partials/order.html]: 1 dot

**Review by SergGrishkov — CHANGES_REQUESTED** [review]

File: src/partials/order.html (hunk 2 above)
Comment (dmeshkovGit) [reply src/partials/order.html]: fixed

File: src/partials/order.html (hunk 1 above)
Comment (dmeshkovGit) [reply src/partials/order.html]: fixed

File: src/partials/order.html (hunk 3 above)
Comment (dmeshkovGit) [reply src/partials/order.html]: done

File: src/partials/order.html (hunk 4 above)
Comment (dmeshkovGit) [reply src/partials/order.html]: done

File: src/partials/order.html (hunk 5 above)
Comment (dmeshkovGit) [reply src/partials/order.html]: done

**Review by SergGrishkov — APPROVED** [review]

**Review by SergGrishkov — APPROVED** [review]

**Pull request merged**